app/com/lynxanalytics/biggraph/graph_operations/DeriveHTMLPython.scala

```diff
@@ -0,0 +1,49 @@
+// Creates an HTML output using Python executed on Sphynx.
+// For security, this output should not be freely controlled by the user.
+// Instead it's used for returning plots encoded in HTML.
```

Comment:
You can see here how torn I am between making this something extremely generic and limiting it to plots. We could call it `DerivePlotPython`. But what if we later add another Python operation that returns HTML? We would have to copy-paste the whole thing. Or I guess we could subclass it. That doesn't sound so bad actually. What do you think?

Reply:
> I would make a subclass if I were you.

👍 If we need another Python+HTML operation later, we'll do that. (I think creating a big automatic graph analysis report is one idea that could use this. We'll see.)